media-tv/mythtv/mythtv-35.0_p20250723.ebuild

```diff
@@ -12,7 +12,7 @@ inherit qmake-utils readme.gentoo-r1 systemd toolchain-funcs user-info
 DESCRIPTION="Open Source DVR and media center hub"
 HOMEPAGE="https://www.mythtv.org https://github.com/MythTV/mythtv"
 if [[ ${PV} == *_p* ]] ; then
-	MY_COMMIT=
+	MY_COMMIT=0a868b015e7346a9156a389acbbe395ec2e1aa24
```

Comment:
Maybe note in the commit message that the snapshot is from the fixes/35 branch, not master.